### PR TITLE
Address supported functionality section for Plugwise

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
This continues on from https://github.com/home-assistant/home-assistant.io/pull/36087 and (too big https://github.com/home-assistant/home-assistant.io/pull/36333) effort

- [x] docs-supported-functions - reworked by platform section for clear structure

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Updated section title from "Entities" to "Supported functionality."
	- Added new subsections for "Climate entities" and "Power and gas entities."
	- Enhanced descriptions for clarity and consistency regarding supported devices and functionalities, including the addition of Aqara Smart Plugs.

### CODE REVIEW

**Comment by CoMPaTech** [comment]
Following the review comment in #36333 to adhere to https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions/ I think the suggested changes are stretching it, so suggestions are welcome. The split between climate and power/gas has been intentional, but it could be one big list (I opted for that split as per lower section of the IQS-rule.

**Comment by CoMPaTech** [comment]
cc @bouwew

**Comment by CoMPaTech** [comment]
@coderabbitai review

**Comment by CoMPaTech** [comment]
@bouwew please review

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -65,13 +65,88 @@ For a thermostat, the active schedule can be deactivated or reactivated via the
...
+  - **Description**: Your netto electricity use at this time, can be negative when producing energy, i.e. though solar panels.
+  - **Gateways**: P1.
+- **P1 Electricity Produced off-peak cumulative**
+  - **Description**: The total produced electricity during off-peak.
+  - **Gateways**: P1.
+
+#### Switches
+
+- **Relay**
+  - **Description**: Turn a plug on or off
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Please use "switch a plug on or off", more suitable to use switch I think.

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -65,13 +65,88 @@ For a thermostat, the active schedule can be deactivated or reactivated via the
...
+A generous number of sensors is provided for your climate setup. Examples include:
+
+- **Setpoint**
+  - **Description**: The setpoint for the current zone (Adam) or generic (Anna).
+- **Indoor temperature**
+  - **Description**: For Anna, Lisa or Jip this will show the temperature measured at the specific thermostat.
+- **Outdoor temperature**
+  - **Description**: The temperature your climate gateway retrieves online.
+- **Outdoor air temperature**
+  - **Description**: If you have an auxiliary device with a temperature sensor.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: The temperature reported by the sensor in the outdoor unit of a heat pump.

**Review by bouwew — APPROVED** [review]
LGTM!

**Review by silamon — APPROVED** [review]

**Comment by CoMPaTech** [comment]
tnx @silamon

**Pull request merged**